packages/gatsby/cache-dir/root.js

```diff
@@ -9,7 +9,10 @@ import {
 } from "./navigation"
 import { apiRunner } from "./api-runner-browser"
 import loader from "./loader"
-import JSONStore from "./json-store"
+import {
+  JSONStoreForPageQueries,
+  JSONStoreForStaticQueries,
```

Comment:
Maybe these could be PageQueryStore and StaticQueryStore (or similar)? I regret including "JSON" in the name of this file :)

Reply:
Great idea, will also probably rename file as well

Reply:
Renamed `json-store.js` to `query-result-store.js` + applied component name change suggestions